edsl.md

```diff
@@ -185,7 +185,11 @@ where `F1 : F2 : F3 : F4` is the (two's complement) byte-array representation of
       requires #range(0 <= DATA < (2 ^Int (SIZE *Int 8)))
       [concrete]
 
-    rule #Ceil(#buf(SIZE, DATA)) => {(0 <=Int SIZE) andBool #rangeBytes(SIZE, DATA) #Equals true}  [anywhere, simplification]
+    rule #Ceil(#buf(@SIZE, @DATA))
```

Comment:
By the way, how is `@SIZE` different from `SIZE`?

Reply:
`SIZE` is an element (singular) variable; the rule with `SIZE` will only match a term where the first argument of `#buf` is known to be defined. `@SIZE` is a set variable, so the rule will match any arguments, and check the `#Ceil` later. It's usually preferred to write simplification rules with set variables.